src/library/librarycontrol.cpp

```diff
@@ -793,6 +816,25 @@ void LibraryControl::slotMoveFocus(double v) {
             QEvent::KeyPress, key, Qt::NoModifier, QString(), false, times});
 }
 
+void LibraryControl::slotMoveTrackForward(double v) {
+    if (v > 0) {
+        slotMoveTrack(1);
+    }
+}
+
+void LibraryControl::slotMoveTrackBackward(double v) {
+    if (v > 0) {
+        slotMoveTrack(-1);
+    }
+}
+
+void LibraryControl::slotMoveTrack(double v) {
+    const auto key = (v < 0) ? Qt::Key_Up : Qt::Key_Down;
+    const auto times = static_cast<unsigned short>(std::abs(v));
+    emitKeyEvent(QKeyEvent{
+            QEvent::KeyPress, key, Qt::AltModifier, QString(), false, times});
```

Comment:
unfortunately emitKeyEvent requires the library to be focused, which would limit the use cases (same with the current navigation controls, see #9294, #11285 and others).

Let's send the key event to `m_pLibraryWidget` directly as this is the only widget that would accept this event (check if is not null)

Reply:
All right, I couldn't test yet, does the code look right?